packages/bento-design-system/src/Table/Config.ts

```diff
@@ -11,4 +11,39 @@ export type TableConfig = {
   hintPlacement: TooltipPlacement;
   cellTooltipPlacement: TooltipPlacement;
   evenRowsBackgroundColor: BentoSprinkles["background"];
+  padding: {
+    header: { paddingX: BentoSprinkles["paddingX"]; paddingY: BentoSprinkles["paddingY"] };
```

Comment:
super minor, but maybe we could avoid some repetition by defining a 

```
type CellPaddingConfig = { paddingX: BentoSprinkles["paddingX"]; paddingY: BentoSprinkles["paddingY"] };
```

and using it around